### ISSUE TITLE
Meaning of batch size here

### ISSUE THREAD

**Issue by btickell** [vincentherrmann/pytorch-wavenet#42]
I am wondering what exactly the batch size means in the context of training this network? The batch dimension is inflated during dilation when running the training code and the dataset is set to shuffle for the training set. Does this not imply that a series of discontinuous time series data is being concatenated together and treated as a single sample?
